spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe "Guests show page", type: :feature do 
+  before(:each) do
+    @hotel_1 = Hotel.create!(name: "Tipton Hotel", location: "Boston, Massachusetts")
+    @room_1 = @hotel_1.rooms.create!(rate: 300, suite: "standard")
+    @guest_1 = @room_1.guests.create!(name: "Zack", nights: 4)
+    GuestRoom.create!(room_id: @room_1.id, guest_id: @guest_1.id)
```

Comment:
Good object setup - might be useful to have more than 1 instance of each object, just to have more stuff to play with.